### DISCUSSION TITLE
LovyanGFX support?

### DISCUSSION THREAD

**Discussion by Roger-random** [Roger-random/ESP_8_BIT_composite#31] [Ideas]
If anyone is interested in using LovyanGFX library (https://github.com/lovyan03/LovyanGFX) instead of AdafruitGFX to their draw color composite output, please describe your usage scenario.

@riraosan is evaluating LovyanGFX for a project, and has created #30 proof of concept adding LovyanGFX support to ESP_8_BIT_Composite.

LovyanGFX users are encouraged to contribute to this discussion of whether it should be merged into ESP_8_BIT_composite.

**Comment by riraosan** [comment]
Hello everyone. Im @riraosan.
I have uploaded a pattern of my LovyanGFX library Panel_CVBS class in action.
Best regards!

- for example

examples/Sprite/PartyParrot/PartyParrot.ino
examples/Sprite/FlashMemSprite/FlashMemSprite.ino
examples/Sprite/MovingCircles/MovingCircles.ino
examples/Sprite/MovingIcons/MovingIcons.ino
examples/Sprite/RotatedZoomSample/RotatedZoomSample.ino
examples/Sprite/ClockSample/ClockSample.ino

https://www.instagram.com/p/CbXvBUovzNE/

https://github.com/riraosan/LovyanGFX/blob/5f2801459d0959ebb0b6507746ec92381b7f21d3/src/lgfx/v1/panel/Panel_CVBS.hpp

**Comment by riraosan** [comment]
Hi. @Roger-random and folks

私はあなたの助言に感謝します。私は次のリポジトリに示すように、LovyanGFXとESP_8_BIT_compositeに依存するかたちで、Panel_CVBSクラスとESP32_8BIT_CVBSユーザークラスをESP32_8BIT_CVBSライブラリとして公開しました。
私はM5GFXライブラリとLovyanGFXのライブラリよりサンプルプログラムを多数転載させていただきました。
どこかのどなたかのなんらかに貢献できましたら幸いです。

I appreciate your advice, I will try to find a solution to my problem. I have published the Panel_CVBS class and ESP32_8BIT_CVBS user class as ESP32_8BIT_CVBS library with dependencies on LovyanGFX and ESP_8_BIT_composite as shown in the following repository.
I have reproduced many sample programs from M5GFX library and LovyanGFX library.
I would be happy if I could contribute to someone somewhere in some way.

https://github.com/riraosan/ESP32_8BIT_CVBS

**Comment by Roger-random** [comment]
Thank you for sharing the results of your hard work!

**Reply by riraosan** [reply]
Thank you. Enjoy! 👍
